repos/base-hw/src/core/include/spec/x86_64/board.h

```diff
@@ -34,4 +34,4 @@ namespace Genode
 	};
 }
 
-#endif /* _CORE__INCLUDE__SPEC__X86__BOARD_H_ */
```

Comment:
Moving x86/board.h to x86_64/board.h is not necessary atm, I will revert it. The constants are not 64bit-specific of course.

If you want to get rid of the x86/x86_64 distinction really depends on your roadmap for base-hw. But I agree that this is definitely a separate issue.